pkg/authorization/registry/clusterpolicy/etcd/etcd.go

```diff
@@ -21,11 +22,12 @@ type REST struct {
 }
 
 // NewStorage returns a RESTStorage object that will work against nodes.
-func NewStorage(s storage.Interface) *REST {
+func NewStorage(optsGetter restoptions.Getter) *REST {
```

Comment:
@liggitt go, so generally it won't be difficult to adapt the Getter to work with generic.RESTOption and we can do it on one place (I think the Getter will have to implement the RESTOption interface).